conda/recipes/dask-cuda/recipe.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  minor_version: ${{ (version | split("."))[:2] | join(".") }}
+  date_string: '${{ env.get("RAPIDS_DATE_STRING") }}'
+  py_version: ${{ env.get("RAPIDS_PY_VERSION") }}
+  py_buildstring: ${{ py_version | version_to_buildstring }}
+  head_rev: ${{ git.head_rev(".")[:8] }}
+
+package:
+  name: dask-cuda
+  version: ${{ version }}
+
+source:
+  path: ../../..
+
+build:
+  noarch: python
+  string: py${{ py_buildstring }}_${{ date_string }}_${{ head_rev }}
+  script:
+    content: |
+      python -m pip install . -vv
+  python:
+    entry_points:
+      - dask-cuda-worker = dask_cuda.cli:worker
+      - dask-cuda-config = dask_cuda.cli:config
+
+requirements:
+  host:
+    - python
+    - pip
+    - rapids-build-backend >=0.3.0,<0.4.0dev0
+    - setuptools >=64.0.0
+  run:
+    - python
+    - click >=8.1
+    - numba >=0.59.1,<0.61.0a0
+    - numpy >=1.23,<3.0a0
+    - pandas >=1.3
+    - pynvml >=12.0.0,<13.0.0a0
+    - rapids-dask-dependency =${{ minor_version }}
+    - zict >=2.0.0
```

Comment:
However it looks like we have expanded this for `rattler-build`

Is there not a way to load this from `pyproject.toml` with `rattler-build`?

Reply:
I don't believe so, currently.  There's not support for the jinja looping constructs to handle splatting out a list of dependencies.  I think updates to the dependency generator are the long-term solution here, now that the conda recipes are in standardized yaml